tests/Functional/Doctrine/Mock/DriverConnectionMock.php

```
@@ -20,47 +20,43 @@
-        return (string) $input;
+        return (string) $value;
     }
 
-    public function exec($statement): int
+    public function exec(string $sql): int
     {
         throw new \BadMethodCallException('Not implemented');
     }
 
-    public function lastInsertId($name = null)
+    public function lastInsertId($name = null): string|int|false
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$name'. (undefined)

(UnusedFormalParameter)